device-types/Nokia/7210-SAS-M-1.yaml

```diff
@@ -0,0 +1,67 @@
+---
+manufacturer: Nokia
+model: 7210 SAS-M-1
+slug: nokia-7210-sas-m-1
+part_number: 3HE04699AAAB01
+u_height: 2
+is_full_depth: false
```

Comment:
Please add the `weight` and `weight_unit` attribute if you have it available

Reply:
Add